docs/docs/tutorial/chapter1/first-page.md

```diff
@@ -43,13 +43,21 @@ const Routes = () => {
 export default Routes
 ```
 
+As long as you have a Home route with path `/`, you'll never see the initial Redwood splash screen again.
```

Comment:
```suggestion
As long as you have a route with path `/`, you'll never see the initial Redwood splash screen again.
```

Saying "Home" implies that the route has to be named that or something, but really any route at `/` will take over the splash screen.